homeassistant/components/efesto/climate.py

```diff
@@ -0,0 +1,290 @@
+
+    @property
+    def fan_mode(self):
+        """Return fan mode."""
+        if self._current_power in FAN_MODES:
+            return self._current_power
+        return FAN_1
+
+    @property
+    def fan_modes(self):
+        """Return the list of available fan modes."""
+        return FAN_MODES
+
+    @property
+    def hvac_action(self):
+        """Return the current running hvac operation if supported.
+
+        Need to be one of CURRENT_HVAC_*.
+        """
+        if self._human_device_status in CURRENT_HVAC_MAP_EFESTO_HEAT:
+            return CURRENT_HVAC_MAP_EFESTO_HEAT.get(self._human_device_status)
+        return CURRENT_HVAC_IDLE
+
+    def turn_off(self):
+        """Turn device off."""
+        try:
+            self.client.set_off()
+        except EfestoError as err:
+            _LOGGER.error("Failed to turn off device (original message: %s)", err)
+
+    def turn_on(self):
+        """Turn device on."""
+        try:
+            self.client.set_on()
+        except EfestoError as err:
+            _LOGGER.error("Failed to turn on device (original message: %s)", err)
+
+    def set_temperature(self, **kwargs):
+        """Set new target temperature."""
+        temperature = kwargs.get(ATTR_TEMPERATURE)
+        if temperature is None:
+            return
+
+        try:
+            self.client.set_temperature(round(temperature))
+        except EfestoError as err:
+            _LOGGER.error("Failed to set temperature (original message: %s)", err)
+
+    def set_fan_mode(self, fan_mode):
+        """Set new target fan mode."""
+        if fan_mode is None:
+            return
+
+        try:
+            self.client.set_power(fan_mode)
+        except EfestoError as err:
+            _LOGGER.error("Failed to set temperature (original message: %s)", err)
+
+    def set_hvac_mode(self, hvac_mode):
+        """Set new target hvac mode."""
+        if hvac_mode == HVAC_MODE_OFF:
+            self.turn_off()
+        elif hvac_mode == HVAC_MODE_HEAT:
+            self.turn_on()
+
+    def update(self):
+        """Get the latest data."""
+        try:
+            device = self.client.get_status()
+        except UnauthorizedError:
+            _LOGGER.error("Wrong credentials for device %s", device)
+            return False
+        except ConnectionError:
+            _LOGGER.error("Connection to %s not possible", self.client.url)
+            return False
+        except EfestoError as err:
+            _LOGGER.error("Error: %s", err)
+            return False
+
+        self._device_status = device.device_status
+        self._current_temperature = device.air_temperature
+        self._target_temperature = device.last_set_air_temperature
+        self._human_device_status = device.device_status_human
+        self._smoke_temperature = device.smoke_temperature
+        self._real_power = device.real_power
+        self._current_power = device.last_set_power
+
+        if self._device_status == 0:
```

Comment:
Use it as a falsy value:
```suggestion
        if not self._device_status:
```